.docs/instalace.md

```diff
@@ -3,21 +3,21 @@
 Aplikace vyžaduje:
 - PHP 8.1
 - MySQL 8
-- Apache
+- Nginx
```

Comment:
Ten na lebedě není, ne?

Reply:
To je celkem jedno, v projektu není nic, co by bylo Apache only

Reply:
Projekt nejede v dockeru ale čistě na serveru. Pro konfiguraci je nginx lepší.